profiling/README.md

```diff
@@ -0,0 +1,19 @@
+# Profiling
+
+The performance tests are designed to hit the APIs using the [Vegeta utility](https://github.com/tsenart/vegeta)
```

Comment:
*Sandbox API?

Reply:
I think it's safe to add a note here to use `brew install vegeta` to install it.

Reply:
You can change it to whatever API endpoint you want, the checked in one happens to be sandbox.